03.bowling/bowling.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+
+score = ARGV[0]
+scores = score.split(',')
+
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+frames_tmp = shots.each_slice(2).to_a
+pins_of_tenth = frames_tmp.slice(9..-1)
+frame_of_tenth = []
+pins_of_tenth.map do |pin|
+  pin = [10] if pin == [10, 0]
+  frame_of_tenth += pin
+end
+frames = frames_tmp.slice(0..8)
+frames.push(frame_of_tenth)
+
+point = 0
+frames.each_with_index do |frame, idx|
```

Comment:
```suggestion
frames.each_with_index do |current_frame, idx|
```

次のフレームやその次のフレームと区別するために、「現在の」と付けてやってもいいかも

Reply:
確かに「何のフレームだ？」って一瞬なりそうですね！
修正します！